sys/kern/vfs_syscalls.c

```diff
@@ -3059,11 +3058,9 @@ getutimes(const struct timeval * __capability usrtvp, enum uio_seg tvpseg,
 		if (tvpseg == UIO_SYSSPACE) {
 			tvp = usrtvp;
 		} else {
-			tmptvp = &tv[0];
-			if ((error = copyin(usrtvp, tmptvp, sizeof(tv))) != 0)
+			if ((error = copyin(usrtvp, tv, sizeof(tv))) != 0)
 				return (error);
-			tvp = (__cheri_tocap struct timeval * __capability)
-			    tmptvp;
+			tvp = tv;
```

Comment:
I think this was from when array decay didn't work correctly.

Reply:
I suspected it was to work around a compiler bug.  There was another case where `&ts[0]` was passed instead of `ts` in some compat code that I changed back to reduce the diff in one of these changes that was probably for the same reason.